docs/Project.toml

```diff
@@ -1,11 +1,16 @@
 [deps]
 Catalyst = "479239e8-5488-4da2-87a7-35f2df7eef83"
+DensityInterface = "b429d917-457f-4dbc-8f4c-0cc954292b1d"
 DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+GR_jll = "d2c73de3-f751-5644-a686-071e5b155ba9"
 Graphs = "86223c79-3864-5bf0-83f7-82e725a168b6"
 JumpProcesses = "ccbc3e58-028d-4f4c-8cd5-9ae44345cda5"
 OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
+PointProcesses = "af0b7596-9bb0-472a-a012-63904f2b4c55"
+StableRNGs = "860ef19b-820b-49d6-a774-d7a799459cd3"
```

Comment:
These all need compat entries.

Reply:
Fixed. I have also updated all the dependencies.